amundsen_application/static/js/components/SearchPage/tests/index.spec.tsx

```diff
@@ -93,40 +113,66 @@ describe('SearchPage', () => {
       expect(props.getPopularTables).toHaveBeenCalled();
     });
 
-    it('calls getSelectedTabByResourceType with correct value', () => {
-      expect(getSelectedTabByResourceTypeSpy).toHaveBeenCalledWith('table');
-    });
-
-    it('calls setState with result of getSelectedTabByResourceType', () => {
-      expect(setStateSpy).toHaveBeenCalledWith({ selectedTab:  mockResourceType });
+    it('calls setState', () => {
+      expect(setStateSpy).toHaveBeenCalledWith({ selectedTab: ResourceType.table });
     });
 
     describe('when searchTerm in params is valid', () => {
+      beforeAll(() => {
+        updatePageUrlSpy.mockClear();
+        const {props, wrapper} = setup({
+          location: {
+            search: '/search?searchTerm=testName&selectedTab=table&pageIndex=1', 
+            pathname: 'mockstr',
+            state: jest.fn(),
+            hash: 'mockstr',
+          }
+        });
+        updatePageUrlSpy = jest.spyOn(wrapper.instance(), 'updatePageUrl');
+        wrapper.instance().componentDidMount();
+      });
       it('calls searchAll', () => {
         expect(searchAllSpy).toHaveBeenCalledWith('testName', mockSearchOptions);
```

Comment:
checking `expect(searchAllSpy).toHaveBeenCalledWith(mockSanitizedUrlParams.term, mockSearchOptions);` will not more accurately test the logic.